src/scroll-to-top.css

```
@@ -0,0 +1,93 @@
+
+@media (max-width: 480px) {
+  .scroll-to-top {
+    width: 40px;
+    height: 40px;
+    font-size: 18px;
+    bottom: 15px;
+    right: 15px;
+  }
+}
+
+@keyframes scrollToTopFadeIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "scrollToTopFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)